app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -99,4 +102,21 @@ def run(zone: nil, role: "automation", object_type: nil, object_id: nil)
 
     run_queue(:zone => zone, :role => role, :object => object) if next_state.present?
   end
+
+  class << self
+    def floe_docker_runner
+      @floe_docker_runner ||= begin
+        if MiqEnvironment::Command.is_podified?
+          token_file = "/run/secrets/kubernetes.io/serviceaccount/token"
+          server     = URI::HTTPS.build(:host => ENV["KUBERNETES_SERVICE_HOST"], :port => ENV["KUBERNETES_SERVICE_PORT"])
+
+          Floe::Workflow::Runner::Kubernetes.new("token_file" => token_file, "server" => server)
+        elsif MiqEnvironment::Command.is_appliance?
+          Floe::Workflow::Runner::Podman.new
+        else
+          Floe::Workflow::Runner::Docker.new
+        end
```

Comment:
@bdunne Since you run podman locally, I recall that we were trying to avoid this kind of logic, instead checking if podman is installed first, then falling back to docker.  Do you recall where / how we did that?  I think it was just a bunch of `which` statements

Reply:
We could add podman, docker, and kubectl to `MiqEnvironment::Command.supports_*` that is where `supports_command?()` is

Reply:
Yeah that would be ok with me

Reply:
https://github.com/ManageIQ/manageiq/pull/22603